frontend/src/metabase/notifications/DashboardSubscriptionsSidebar/tests/common.unit.spec.ts

```diff
@@ -202,6 +202,11 @@ describe("DashboardSubscriptionsSidebar", () => {
       // Create the first subscription
       await userEvent.click(screen.getByRole("button", { name: "Done" }));
 
+      // (EMB-1099) The subscription channel options sidebar shouldn't be shown
```

Comment:
These assertions work. You can try reverting the code, and these 2 tests should fail.